crates/nix_rs/src/flake/schema.rs

```diff
@@ -64,6 +78,17 @@ impl FlakeSchema {
             checks: pop_per_system_tree(output, "checks"),
             apps: pop_per_system_tree(output, "apps"),
             formatter: pop_leaf_type(output, "formatter"),
```

Comment:
Any reason why we can't use schemas for `packages`, `checks`, `apps`, `formatter` as well?

At which point, we may even be able to refactor things a bit.

Reply:
> we may even be able to refactor things a bit.

Thus, enabling new output types when the user uses their own schema, for whatever reason, and have it propagate to output of `om show` (currently we hardcode the schema types to be rendered). Something to think ahead ...